aie_runtime_lib/AIE2/vec_math.h

```diff
@@ -229,4 +229,21 @@ getSigmoidBf16(v32bfloat16 in) {
   aie::vector<bfloat16, 32> out = msc0_acc.to_vector<bfloat16>();
   return (v32bfloat16)out;
 }
+
+inline __attribute__((always_inline)) v16bfloat16 getCeilBf16(v16bfloat16 in) {
+  aie::vector<int32, 16> in_int32 = bfloat16_to_int(
```

Comment:
This function will compute incorrect results if abs(in) > 0x1.p31
The range of bfloat16 is -1.fep127 .. +1.fep127.
For input values that are < -2^31 or > 2^31-1 we should return the input as-is.